kdluu/ptm_tool_app
# changes to show invalid user/password on login page

## Changes
- registration page for tool-tip 
- login page to show error for invalid user or password

### How To test
- login with invalid user or password on login page should show error
- tool tip should show when error on registration jsp
